service/status.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf8
+# Copyright 2016, 2021 John J. Rofrano. All Rights Reserved.
```

Comment:
We probably want our own copyright header and not John's :)

Reply:
lol didn't notice that